src/main/steps/applicant1/hub-page/joint/content.ts

```diff
@@ -253,6 +250,19 @@ const cy: typeof en = ({ isDivorce, userCase, partner, isApplicant2 }: CommonCon
     link: `${isApplicant2 ? `${APPLICANT_2}'/how-to-finalise'` : '/how-to-finalise'}`,
     linkText: ' as a sole applicant',
   },
+  applyForFinalOrderAfterSwitchToSole: {
+    line1: `You can now apply for a ‘final order’ as a sole applicant. If it’s granted then your ${
+      isDivorce ? 'marriage' : 'civil partnership'
```

Comment:
maybe we should move this to common content, like partner

Reply:
https://tools.hmcts.net/jira/browse/NFDIV-3183